File: php/6kyu - B.php
<?php

//Link -> https://www.codewars.com/kata/59df2f8f08c6cec835000012

function meeting($s) {
    $guests = explode(";", strtoupper($s));
    $fullName = [];
    $newS = "";

    foreach($guests as $item){
        array_push($fullName, array_reverse(explode(":", $item)));
    }

    sort($fullName);

    foreach($fullName as $item){
        $newS .= "(".$item[0].", ".$item[1].")";
    }

    return $newS;
}

?>

File: python/6kyu - B.py
# Link -> https://www.codewars.com/kata/5264d2b162488dc400000001

def spin_words(sentence):
    words = sentence.split()
    spin = ''

    for i in words:
        if len(i) >= 5:
            spin += i[::-1] + ' '
        else:
            spin += i + ' '

    spin = spin.rstrip()

    return spin
